api/services/PassportService.js

```diff
@@ -203,11 +203,12 @@ module.exports = class PassportService extends Service {
 
     criteria[fieldName] = identifier
 
-    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports', findOne: true})
+    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports'})
```

Comment:
I was wondering why remove the findOne ? trailpack-mongoose doesn't support it ? :(